client/js/jsonedit.js

```diff
@@ -2198,6 +2197,10 @@ function jeToggle() {
     $('body').classList.add('jsonEdit');
     if(jeWidget && !widgets.has(jeWidget.id))
       jeEmpty();
+    if(jeDebugViewing) {
+      jeCallCommand(jeCommands.find(o => o.id == 'je_toggleDebug'));
+      jeShowCommands()
+    }
```

Comment:
Doesn't `jeEmpty` already do that? Maybe just move it up a bit to avoid a second call?

Reply:
I considered that. But right now, `jeEmpty` is being called only if there was a widget being displayed before that has now been deleted. So we could make the `jeEmpty` call unconditional and move the new code up, but I'm not sure that buys much if anything.

Reply:
I'm just careful after the optimization PR. Most of the slowdowns came from just carelessly calling the same function again and again.

Both of those are in very narrow conditions though so just keep it as it is.